src/40-Documentation.md

```diff
@@ -0,0 +1,5 @@
+## 4.0 Documentation
+There are multiple types of documentation. 
```

Comment:
This section is **TODO**

- [ ] Code Documentation Section
- [ ] In Code Documentation Section
- [ ] Code Section